code/game/objects/items/weapons/storage/uplink_kits.dm

```diff
@@ -83,6 +83,16 @@
 				new /obj/item/weapon/fireaxe(src)
 				return
 
+			if("hotline")
+				new /obj/item/clothing/under/jacketsuit(src)
+				new /obj/item/clothing/head/helmet/biker(src)
+				new /obj/item/clothing/shoes/mime/biker(src)
+				new /obj/item/clothing/gloves/bikergloves(src)
+				new /obj/item/clothing/mask/gas/voice(src)
+				new /obj/item/weapon/kitchen/utensil/knife/large/butch/meatcleaver(src)
```

Comment:
This is the syndie meat cleaver you can normally get as a chef traitor. I'd consider three of them too powerful for one syndie bundle and because of the way the game is you'd have to be nuts to ever actually throw one. Shit sucks but there you go.